pkg/volume/plugins.go

```diff
@@ -714,16 +714,16 @@ func (pm *VolumePluginMgr) FindPluginByName(name string) (VolumePlugin, error) {
 	matchedPluginNames := []string{}
 	matches := []VolumePlugin{}
 	for k, v := range pm.plugins {
-		if v.GetPluginName() == name {
+		if k == name {
```

Comment:
``pm.plugins`` type is ``map[string]VolumePlugin``,  so I think the key is unique?
Is this should be cleaner?
```
	if v, ok := pm.plugins[name]; ok {
		matchedPluginNames = append(matchedPluginNames, name)
		matches = append(matches, v)
	}
```

Reply:
Right. thanks, I would change the code.